.github/workflows/storage.yml

```diff
@@ -156,3 +156,37 @@ jobs:
       run: scripts/setup_bundler.sh
     - name: PodLibLint Storage Cron
       run: scripts/third_party/travis/retry.sh scripts/pod_lib_lint.rb FirebaseStorage.podspec --platforms=${{ matrix.target }} --use-static-frameworks --skip-tests
+
+  doc-diff:
```

Comment:
Should we consolidate the steps into a script so we can inject less into each product's yml file?